svc-bie-kafka/build.gradle

```diff
@@ -32,7 +32,7 @@ dependencies {
 
   // RabbitMQ
   implementation 'org.springframework.boot:spring-boot-starter-amqp'
-  implementation 'org.springframework.kafka:spring-kafka'
+  implementation 'org.springframework.kafka:spring-kafka:2.9.11'
```

Comment:
Set this version in https://github.com/department-of-veterans-affairs/abd-vro/blob/1ab3968ce669af1de76d8c7238de8d4e1b1cfed2/gradle-plugins/src/main/groovy/shared.java.vro-dep-constraints.gradle#L27